Watchman.DomainModel/Settings/Commands/Handlers/ChangeConfigurationCommandHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Threading.Tasks;
+using Watchman.Cqrs;
+using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Settings.Commands.Handlers
+{
+    internal class ChangeConfigurationCommandHandler : ICommandHandler<ChangeConfigurationCommand>
+    {
+        private readonly ISessionFactory _sessionFactory;
+
+        public ChangeConfigurationCommandHandler(ISessionFactory sessionFactory)
+        {
+            this._sessionFactory = sessionFactory;
+        }
+
+        public async Task HandleAsync(ChangeConfigurationCommand command)
```

Comment:
github actions pokazuje mi tu jakieś warningi

Reply:
`This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.`